deno/payloads/v6/channel.ts

```diff
@@ -131,7 +131,7 @@ export interface APIMessageApplication {
 }
 
 /**
- * https://discord.com/developers/docs/resources/channel#message-object-message-reference-structure
+ * https://discord.com/developers/docs/resources/channel#message-reference-object-message-reference-structure
```

Comment:
v6 and v8 don't get updates

Reply:
I thought it would be good, as they are links anyway. Be aware that it won't work for people using these vesions.
But if you disagree I can revert it.